modules/cbmarkdown/models/Processor.cfc

```diff
@@ -39,7 +30,22 @@ component accessors="true" singleton{
 	* @txt The markdown text to convert
 	*/
 	function toHTML( required txt ){
-		return trim( variables.processor.process( trim( arguments.txt ), variables.config ) );
+		var document = variables.parser.parse( trim( arguments.txt ) );
+		return trim( variables.renderer.render( document ) );
+	}
+
+	private function createOptions() {
+		var TE = jl.create( "com.vladsch.flexmark.ext.tables.TablesExtension" );
+		return jl.create( "com.vladsch.flexmark.util.options.MutableDataSet" )
+			.init()
```

Comment:
All these options should be available via options in the ModuleConfig as settings.  I am not sure what other options are available, but I would document them in the settings for the module.